backend/tests/regression/answer_quality/file_uploader.py

```diff
@@ -52,6 +52,7 @@ def upload_test_files(zip_file_path: str, env_name: str) -> None:
 
```

Comment:
Not here! but for the tests, can you mock the call to get search settings just to make the failing test pass?